test/com/google/javascript/jscomp/parsing/NewParserTest.java

```diff
@@ -2657,7 +2657,44 @@ public void testImport() {
     parse("import {x, y} from './someModule'");
     parse("import {x as x1, y as y1} from './someModule'");
     parse("import {x as x1, y as y1, } from './someModule'");
+    parse("import {default as d} from './someModule'");
+    parse("import d, {x as x1, y as y1} from './someModule'");
     parse("import * as sm from './someModule'");
+
+    parseError("import class from './someModule'",
+            "cannot use keyword 'class' here.");
+    parseError("import * as class from './someModule'",
+            "'identifier' expected");
+    parseError("import {a as class} from './someModule'",
```

Comment:
If I understand correctly {a as class} is invalid (makes sense as it would put a var called 'class' into the module scope) but {class as a} is valid. I don't see tests of this valid case ({someKeyword as nonKeyword}) except for `default`.